### PR TITLE
Complete using statements and qualified decorators

### PR DESCRIPTION
**Pull request by nguerrera**
Fix https://github.com/Azure/cadl-azure/issues/1034

completioni

### CODE REVIEW

File: packages/compiler/core/binder.ts
Comment (nguerrera) [inline packages/compiler/core/binder.ts:143]: I can clean this up after the refactor merges, which I believe will add ability to get proper namespace symbol. I'll wait for that PR to merge before merging this one to avoid unnecessary conflict.

File: packages/compiler/core/checker.ts
Comment (nguerrera) [inline packages/compiler/core/checker.ts:115]: Undo.

**Review by timotheeguerin — APPROVED** [review]

File: packages/compiler/core/binder.ts
Comment (bterlson) [reply packages/compiler/core/binder.ts:143]: Storing the namespace name on the symbol shouldn't be necessary post-refactor - once you have a symbol, you can walk its parents to get the FQI.

Comment (nguerrera) [reply packages/compiler/core/binder.ts:143]: Yep, that's my plan :)

**Comment by nguerrera** [comment]
@timotheeguerin Mind reviewing this again? The merge was non-trivial and I had to change it significantly.

File: packages/compiler/core/checker.ts
Comment (timotheeguerin) [inline packages/compiler/core/checker.ts:1099]: nit: triple `!!!` 
```suggestion
        return !(sym.flags & SymbolFlags.Function);
```

**Review by timotheeguerin — APPROVED** [review]

**Pull request merged**
